casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -140,7 +140,7 @@ object Genesis {
       wallets     <- getWallets[F](walletsFile, maybeWalletsPath)
       bonds       <- getBonds[F](bondsFile, numValidators, genesisPath)
       timestamp   <- deployTimestamp.fold(Time[F].currentMillis)(_.pure[F])
-      initial     = withoutContracts(bonds = bonds, timestamp = 1L, version = 0L, shardId = shardId)
+      initial     = withoutContracts(bonds = bonds, timestamp = 1L, version = 1L, shardId = shardId)
```

Comment:
is the `version=1L` always the same as `private val version = 1L` in `MultiParentCasperImpl.scala`? 

If that's the case then we should have a constant in order to make sure that we use the same value everywhere.

Reply:
I'll address this in the on dev branch

Reply:
And here is the ticket for it so that I don't forget https://rchain.atlassian.net/browse/RHOL-1015